src/functions/process-stream-event.ts

```diff
@@ -58,7 +58,7 @@ export const processStreamEvent: Handler = async (
       );
       if (tableName.includes('flat-tech-records')) {
         transformTechRecord(dynamoRecord as _Record);
-        debugLog(`Dynamo Record after transformation: ${dynamoRecord}`);
+        debugLog(`Dynamo Record after transformation: ${JSON.stringify(dynamoRecord)}`);
```

Comment:
What's the need to log out the entire dynamoDB record, i ask as that record is likely to be huge which might result in it becoming difficult to debug.

Reply:
The debug log was already there, it just was not JSON stringifying it. There are a lot of debug logs in here, but obviously they only log when the env variable is added, which would only happen in PROD if we had a serious issue, where knowing the details of the tech record would be helpful.